buzz_el/entity_matcher/entity_matcher.py

```diff
@@ -60,24 +45,20 @@ def __init__(
             The spaCy model to use in internal spaCy components.
         ignore_case : Optional[bool], optional
             Whether to ignore case, by default True.
-        min_r : Optional[int], optional
-            TODO, by default None
-        fuzzy_funct : Optional[str], optional
-            The fuzzy matching function to use, by default None.
+        use_fuzzy : Optional[bool], optional
+            Whether to use fuzzy matching, by default False.
         """
         self.spacy_model = spacy_model
         self.kg = knowledge_graph
         self.ignore_case = ignore_case
-
-        self._string_matcher_config = {"spans_key": "string"}
-        if self.ignore_case:
-            self._string_matcher_config["phrase_matcher_attr"] = "LOWER"
+        self.use_fuzzy = use_fuzzy
 
         self._string_matcher = None
-        self._fuzzy_matcher = None
-
         self.build_string_matcher()
-        # self.build_fuzzy_matcher()
+
+        self._fuzzy_matcher = None
+        if self.use_fuzzy:
+            self.build_fuzzy_matcher()
 
     def __call__(self, doc: Doc) -> Doc:
```

Comment:
If the fuzzy matcher is not None, I would just run the fuzzy matcher. The fuzzy matcher will always match at least what the string matcher does match. Isn't it?

Reply:
Oh yes ok, I didn't understand that in what we said and what we wrote, but it suits me fine I'll make the change.